specifyweb/frontend/js_src/lib/components/DataModel/businessRules.ts

```diff
@@ -158,38 +163,41 @@ export class BusinessRuleManager<SCHEMA extends AnySchema> {
   private async checkUnique(
     fieldName: string & keyof SCHEMA['fields']
   ): Promise<BusinessRuleResult<SCHEMA>> {
-    const scopeFields =
-      this.rules?.uniqueIn === undefined
-        ? []
-        : this.rules?.uniqueIn[
-            this.resource.specifyTable.getField(fieldName)
-              ?.name as TableFields<SCHEMA>
-          ] ?? [];
-    const results: RA<Promise<BusinessRuleResult<SCHEMA>>> = scopeFields.map(
-      async (uniqueRule) =>
-        typeof uniqueRule === 'object'
-          ? this.uniqueIn(uniqueRule.field, [
-              fieldName,
-              ...uniqueRule.otherFields,
-            ])
-          : this.uniqueIn(uniqueRule, [fieldName])
+    const rules =
+      getCache('businessRules', 'uniqueRules')![
```

Comment:
Is cache still necessary?

Reply:
Cache was removed, so actually this was not fetching any rules at all! 
Fixed in https://github.com/specify/specify7/pull/4215/commits/7e2c32db487e57986fb9a5302fc102273dbf37db